manim/mobject/opengl_geometry.py

```diff
@@ -0,0 +1,897 @@
+import numpy as np
+
+from ..constants import *
+from ..utils.color import *
+from ..mobject.mobject import Mobject
+from ..mobject.types.opengl_vectorized_mobject import OpenGLVGroup
+from ..mobject.types.opengl_vectorized_mobject import OpenGLVMobject
+from ..mobject.types.opengl_vectorized_mobject import OpenGLDashedVMobject
```

Comment:
try making these import statements into one.
```suggestion
from ..mobject.types.opengl_vectorized_mobject import (OpenGLDashedVMobject,
                                                       OpenGLVGroup,
                                                       OpenGLVMobject)
```